samples/gltf_viewer.cpp

```diff
@@ -232,6 +236,7 @@ static int handleCommandLineArguments(int argc, char* argv[], App* app) {
         { "settings",        required_argument,    nullptr, 't' },
         { "split-view",      no_argument,          nullptr, 'v' },
         { "vulkan-gpu-hint", required_argument,    nullptr, 'g' },
+        { "screenshot-as-tiff", no_argument,       nullptr, 'd' },
```

Comment:
You should preserve the alignement here, also: any reason to not make TIFF the default? PPM was chosen just because it's easy to output I think, but I doubt there's a good reason to keep both formats?